api/app/signals/apps/questionnaires/models/question.py

```diff
@@ -48,3 +83,15 @@ def save(self, *args, **kwargs):
     @property
     def field_type_class(self):
         return get_field_type_class(self)
+
+    def get_field_type(self):
+        kwargs = {}
+        if self.multiple_answers_allowed and self.additional_validation and 'answers' in self.extra_properties:
```

Comment:
I think there is a small error here:
`and 'answers' in self.extra_properties`
should probably be:
`and 'answers' in self. additional_validation `

(Related see comments in: `api/app/signals/apps/questionnaires/tests/services/test_answer.py` )